styles/styles.css

```diff
@@ -10,6 +10,14 @@
  * governing permissions and limitations under the License.
  */
 
+@font-face {
```

Comment:
I added this in https://github.com/hlxsites/elco-aerotop-sg/pull/25

Reply:
maybe you didn't merge main right?

Reply:
I rebased by mistake. I think merging this PR won't create any problem